Google.PowerShell.IntegrationTests/Provider/Storage.Provider.Tests.ps1

```diff
@@ -71,6 +71,15 @@ Describe "Storage Provider"{
         Compare-Object $gcsContents $localContents | Should BeNullOrEmpty
     }
 
+    It "Should create file using set-content" {
+        cd gs:\$bucketName\$folderName
+        rm $fileName
+        Set-Content $fileName -Value $content1
+        Sleep 1
```

Comment:
Same below. Did you add the `Sleep` on line 144 because the test failed without it? Or was it "just in case"? If it was the latter, let's remove it until it is a problem.